### PR TITLE
EE dev docs

### LINKED ISSUE #1741: Draft local dev docs using docker
Provide doc in the ee/clickhouse directory and posthog docs  that details how to get started on clickhouse

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

Here's a WIP `.md` file where I'll gather wisdom about getting the EE version running in all its glory. 

This should eventually go to the `.com` repo, though for now having it close to the code feels right.

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: ee/DEV.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Getting it up and running
+`docker-compose -f ee/docker-compose.ch.yml up`
+
+## Fixing broken frontend build
+`docker run -it ee_web yarn build`
```
Comment (fuziontech) [inline ee/DEV.md:5]: don't need -it here but works all the same

File: ee/DEV.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Running Python + Webpack locally
+- Run all the services
+  - Stop local kafka, clickhouse and zookeeper instances if you have them
+  - Same for redis, though it doesn't really matter much
+  - `docker-compose -f ee/docker-compose.ch.yml up db redis zookeeper kafka clickhouse`
+  - Add to `/etc/hosts`:
+    - `127.0.0.1       kafka` (setting KAFKA_URL later doesn't seem to work)
```
Comment (fuziontech) [inline ee/DEV.md:13]: nicely done. This is a bug/feature with how the consumer picks up hosts for kafka from zookeeper.

**Review by fuziontech — APPROVED** [review]
🚢 it

**Comment by yakkomajuri** [comment]
Once this is reviewed here I'll add it to our Handbook under Engineering

**Comment by mariusandra** [comment]
Thanks @yakkomajuri ! I think it's good to add it to the docs now!

**Pull request closed without merging**